style.css

```diff
@@ -0,0 +1,154 @@
+* {
+    box-sizing: border-box;
+    margin: auto;
+    padding: 0;
+  }
+
+html {
+    background-color: rgb(226, 226, 226); 
+}
+
+body {
+    background-color:  rgba(230, 214, 255, 0.288);
+    max-width: 65%;
+    }
+
+.header-image {
+    border-radius: 50%;
+    width: 10%;
+    height: 10%;
+    float: left;
+    padding: 10px
+}
+
+.social-media-links {
+    float: right;
+    border-radius: 50%;
+    padding-right: 10px;
+}
+
+.header-image2 {
+    width: 90%;
+    padding: 0 20px 0 20px;
+    height: 209px;
+    display: block;
+}
+
+.header, h1 {
+    /* float:left; */
+    padding-top: 10px;
+}
+
+.clearfix {
+    clear:both;
+}
+
+.menu {
+    padding-top: 5px;
+}
+
+header {
+    padding-top: 20px;
+}
+
+.body h2 {
+    font-size: 2rem;
+    padding-left: 5px;
+}
+
+.body h3 {
+    padding-left: 5px;
+}
+
+h4 {
+    padding-bottom: 17px;
+    padding-left: 5px;
+}
+
+.subheader {
+    margin-left: 2%;
+    height: 185px;
+    border: 7px solid black;
+    float: left;
+    border-radius: 50%;
+    padding: 35px;
+    margin-bottom: 3%;
+    background: linear-gradient(rgba(173, 103, 22, 0.397),rgba(30, 9, 110, 0.5)),
+     no-repeat center/cover url('https://image.ibb.co/dU2Bdc/headshot.png');
+    color: whitesmoke;
+    text-align: center;
+}
+
+.subheader2 {
+    margin-left: 2%;
+    border: 7px solid black;
+    border-radius: 50%;
+    padding: 35px;
+    margin-bottom: 3%;
+    float: left;
+    background: linear-gradient(rgba(173, 103, 22, 0.397),rgba(30, 9, 110, 0.5)),
+     no-repeat center/cover url('https://image.ibb.co/hhptPH/websiteimage.png');
+    height: 185px;
+    color: whitesmoke;
+    text-align: center;
+    
+}
+
+.subheader3 {
```

Comment:
You have a lot of repeated CSS with these subheaders. You should use a common class for all of them that has the CSS they have in common and add another class on each for their differences.